sql/moz-fx-data-shared-prod/telemetry/firefox_use_counters/view.sql

```diff
@@ -0,0 +1,22 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.firefox_use_counters`
```

Comment:
I'm not sure about calling this "firefox_use_counters" because it has desktop and fenix data. Maybe "firefox_desktop_fenix_use_counters".

The only thing is that if other use counters are added in the future, the view would have to be renamed

Reply:
Do you think a more generic "use_counters" would be better, so we can add additional ones in the future?

Reply:
Updated!